paddle/trainer/tests/sample_trainer_config_opt_a.conf

```diff
@@ -12,32 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#Todo(luotao02) This config is only used for unitest. It is out of date now, and will be updated later.
+from paddle.trainer_config_helpers import *
 
 ################################### Data Configuration ###################################
 TrainData(ProtoData(files = "train.list"))
 ################################### Algorithm Configuration ###################################
-Settings(
-    learning_rate_decay_a = 0.0,
-    learning_rate_decay_b = 0.0,
-    learning_rate = 1e-03,
-    batch_size = 1000,
-    algorithm = 'sgd',
-    num_batches_per_send_parameter = 1,
-    num_batches_per_get_parameter = 1,
-    learning_method='sparse_momentum',
-)
-default_momentum(0.5)
+settings(batch_size = 1000,
+         learning_method = MomentumOptimizer(Momentum=0.5, sparse=True))
```

Comment:
is `Momentum`?

https://github.com/baidu/Paddle/pull/126/files#diff-77565d531167fc12792a7aaedf18a243R106 code shows it is `momentum`.

Is this file pass unittest?